src/site/content/en/fast/codelab-extract-and-inline-critical-css/index.md

```diff
@@ -0,0 +1,198 @@
+
+<table>
+    <th>Option</th>
+    <th>Type</th>
+    <th>Explanation</th>
+  <tr>
+    <td><code>base</code></td>
+    <td>string</td>
+    <td>The base directory for your files.</td>
+  </tr>
+  <tr>
+    <td><code>src</code></td>
+    <td>string</td>
+    <td>HTML source file.</td>
+  </tr>
+  <tr>
+    <td><code>dest</code></td>
+    <td>string</td>
+    <td>The target for the output file. If CSS is inlined the output file is HTML. If not, the output is a CSS file.</td>
+  </tr>
+  <tr>
+    <td><code>width</code>, <code>height</code></td>
+    <td>numbers</td>
+    <td>Viewport width and height in pixels.</td>
+  </tr>
+  <tr>
+    <td><code>dimensions</code></td>
+    <td>array</td>
+    <td>Contains objects with width and height properties. These objects represent the viewports you want to target with above-the-fold CSS. If you have media queries in your CSS, this allows you to generate critical CSS that covers multiple viewport sizes.</td>
+  </tr>
+  <tr>
+    <td><code>inline</code></td>
+    <td>boolean</td>
+    <td>When set to true, the generated critical CSS is inlined in the <head> of the HTML source file.</td>
+  </tr>
+  <tr>
+    <td><code>minify</code></td>
+    <td>boolean</td>
+    <td>When set to true, Critical minifies the generated critical CSS. Can be omitted when extracting critical CSS for multiple resolutions because Critical automatically minifies it to avoid duplicate rule inclusion.</td>
+  </tr>
+</table>
+
+
+Below is a configuration example for extracting critical CSS for multiple resolutions. Add it to `critical.js` or play around and tweak the options.
+
+```js/1-14/
+critical.generate({
+  base: 'public/',
+  src: './index.html',
+  dest: './index.html',
+  inline: true,
+  dimensions: [
+    {
+      height: 500,
+      width: 300,
+    },
+    {
+      height: 720,
+      width: 1280,
+    },
+  ]
+}, (err, output) => {
+  if (err) {
+    console.error(err);
+  } else if (output) {
+    console.log('Generated critical CSS');
+  }
+});
+```
+
+In this example, `index.html` is both the source file and the destination file because the `inline` option is set to true. Critical first reads the HTML source file, extracts critical CSS and then overwrites `index.html` with critical CSS inlined in the `<head>`. 
+
+`dimensions` array has two viewport sizes specified: 300 x 500 for extra small screens and 1280 x 720 for standard laptop screens.
+
+`minify` option is omitted because Critical automatically minifies the extracted CSS when there are multiple viewport sizes specified.
+
+### Run Critical
+
+Add Critical to your scripts in `package.json`:
+
+```js/2-2/
+scripts: {
+    "start": "node server.js",
+    "critical": "node critical.js"
+  }
+```
+
+Click **Tools** > **Logs** > **Console**.
+
+To generate critical CSS, in the console, run:
+
+```
+pnpm run critical
```

Comment:
I think in your original glitch you can open the console and do:

```
enable-npm
```

Then folks can do `npm run critical`. I think this should work even if they remix it. Give it a shot and let me know if it works?

Reply:
It worked, I'll edit the instructions. 

I also tried running an npm command on a fresh Glitch starter app, it worked and it didn't even show that "please use pnpm" note 🤔